docs/contributors/coding-guidelines.md

```diff
@@ -174,7 +174,7 @@ In general, avoid backslash-escaping quotes:
 
 ```js
```

Comment:
It looks like this got changed automatically by Prettier, so next time someone edits this file and saves, the problem might be brought back. Ironically, our tooling is ruining the bad example by turning it into a good example. I think adding a `<!-- prettier-ignore -->` before the code block will prevent that from happening again.

```suggestion
<!-- prettier-ignore -->
```js
```

See also: https://prettier.io/docs/en/ignore.html

Reply:
Thank you. I will check if the prettier-ignore tag is the proper solution and submit another patch.

Reply:
I have just tested it, and I can confirm that @ZebulanStanphill is right 💯